zebra/zebra_mpls_vty.c

```diff
@@ -295,9 +295,45 @@ static int zebra_mpls_config(struct vty *vty)
 	write += zebra_mpls_write_lsp_config(vty, zvrf);
 	write += zebra_mpls_write_fec_config(vty, zvrf);
 	write += zebra_mpls_write_label_block_config(vty, zvrf);
+	write += zebra_mpls_write_recursive_seamless_config(vty, zvrf);
 	return write;
 }
 
+/* Enable seamless MPLS */
+DEFUN (mpls_fec_recursive_seamless,
```

Comment:
DEFPY now, please, and you can use one block that includes the 'no' form.

Reply:
Response: Understood, it is going to be addressed.